buildSrc/src/main/kotlin/Java9Modularity.kt

```diff
@@ -24,85 +19,84 @@ object Java9Modularity {
             logger.warn("No Kotlin JVM targets found, can't configure compilation of module-info!")
         }
         jvmTargets.forEach { target ->
+            val artifactTask = tasks.getByName<Jar>(target.artifactsTaskName) {
+                if (multiRelease) {
+                    manifest {
+                        attributes("Multi-Release" to true)
+                    }
+                }
+            }
+
             target.compilations.forEach { compilation ->
```

Comment:
Won't this also affect `test` compilation?

Reply:
Yes, it always did affect the `test` compilation (in case you want a black-box test where the `test` compilation is a module of its own. But it only actually does something if a `module-info.java` is present in the test sources..